index.html

```diff
@@ -53,90 +50,159 @@ <h3>I'm a Software Developer</h3>
+                    <li>javascript</li>
+                    
+                </ul>
+            </div><!--greyout end-->
+            <div class="buttondecor">
+                <p>Not Active</p>
+            </div></a>
+        </div><!--site1 end-->
+        <div class="site site5">
+         <a href="#portfolio"><div class="greyout">
+                <div class="description">
+                    <div class="title">Callas Prize</div>
+                    <p class="text">The advertisement and application process to enter the Callas Prize competition, which was supposed to take place at the Alte Oper Frankfurt for the finals.  Unfortunately cancelled due to COVID.</p>
+                </div>
+                <ul class="items lang">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                    
+                </ul>
+            </div><!--greyout end-->
+            <div class="buttondecor">
+                <p>Not Active</p>
+            </div></a>
+        </div><!--site1 end-->
+        <div class="site site6">
+         <a href="https://philipmodinos.ueuo.com/" rel=”noopener”><div class="greyout">
+                <div class="description">
+                    <div class="title">Singing Portfolio</div>
+                    <p class="text">My personal singing portfolio and also the first site I ever coded.  In it's initial form it was merely a list, but later I created an interesting, albeit somewhat glitchy, navbar system.</p>
+                </div>
+                <ul class="items lang">
+                    <li>html</li>
+                    <li>css</li>
+                </ul>
+            </div><!--greyout end-->
+            <div class="buttondecor">
+                <p>See Project</p>
+            </div></a>
+        </div><!--site1 end-->
         
-    </div><!--container end-->
+    </div><!--container2 end-->
+    
 </section><!--page2 end-->
 <br> <br> <br>
+<a name="about"></a>
 <section class="page3">
     <div class="container">
         <br> <br> 
-        <h3 class="items title1">About me</h3>
-        <p class="items text1">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+        <h3 class="title">About me</h3>
+        <p class="text">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
         </p>
-        <div class="items buttontypeA">
+        <div class="buttontypeA">
             <a href="#">Get My Resume</a>
         </div>
-        <div class="items graphic">
-            <img src="images/graphic.png" alt="">
-        </div>
-        <div class="items hr">
+        <div class="hr">
             <hr>
```

Comment:
kindly use prettier or any vscode extension to ident your code, It will help you organize your code better